src/__tests__/isValidCommitMessage.test.ts

```diff
@@ -7,10 +7,14 @@ test("should be able to correctly validate the commit message", () => {
     expect(isValidCommitMessage("fix: menu must open on shortcut press")).toBe(true);
     expect(isValidCommitMessage("something: should not work")).toBe(false);
     expect(isValidCommitMessage("fixes something")).toBe(false);
-    expect(isValidCommitMessage("🚧 fix: menu must open on shortcut press")).toBe(true);
+    expect(isValidCommitMessage("🚧 fix: menu must open on shortcut press")).toBe(false);
     expect(isValidCommitMessage("fix(menus): menu must open on shortcut press")).toBe(true);
-    expect(isValidCommitMessage("🚧 fix(menus): menu must open on shortcut press")).toBe(true);
+    expect(isValidCommitMessage("🚧 fix(menus): menu must open on shortcut press")).toBe(false);
```

Comment:
I feel that adding emojis to commit messages should be fine as long as the rest of the commit messages adheres to the conventional commits semantics.